### ISSUE TITLE
no such app inoperable on moto g, android 4.4.4

### ISSUE THREAD

**Issue by obidm** [mattt/rack-heroku-no-such-app#2]
after install of nsa 0.6.00 and permitting the local proxy configuration, no data services worked on my moto g android 4.4.4 - neither 3g nor wifi would communicate with any services. rebooting didn't help. I uninstalled NSA, deleted the SandroProxy certificate, and all went back to normal. Thanks.

**Comment by mattt** [comment]
I don't think this is what you're looking for. This is a Ruby web server middleware component. I couldn't imagine how this might be run on an Android device.

**Comment by obidm** [comment]
I apologize.

Sorry, sometimes it's hard to follow ownership on github. For me, anyway.

Doug Simmons
